docs/getting_started.md

```diff
@@ -44,6 +44,18 @@ object MyApp extends App {
     } yield ()
 }
 ```
+`run` should return a ZIO value which has all its errors handled,  
+which, in zio parlance, is aka an unexceptional ZIO value.  
+
+One way to do that, is to invoke `fold` over a ZIO value, to get an unexceptional ZIO value  
+That requires two handler functions, one `eh: E => B`, other `ah: A => B`  
+If myAppLogic fails, `eh` will be used to get from e:E to b:B  
+if it suceeds, `ah` will be used to get from a:A to b:B  
```

Comment:
Maybe for a nicer flow start all sentences with an uppercase letter and end with a dot e.g. "If it succeeds, `ah` will be used to get from a:A to b:B."

Reply:
This is a bit custom philosophy. 'Fullstop capital letter' sentence breaking pattern is used if it's deemed to be more meaningful. But here, I didn't intend to break the sentence, and yet give a pause, without making the sentence look too long. Hence ..